bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -354,10 +352,13 @@ protected void refreshStatus() {
             if (refreshSettings || (scheduledUpdates > 0) || (skipUpdate % skipCount == 0)) {
                 if (!profile.isInitialized() || ((thingStatus == ThingStatus.OFFLINE))
                         || (thingStatus == ThingStatus.UNKNOWN)) {
+                    if (getThing().getStatusInfo().getStatusDetail() == ThingStatusDetail.DISABLED) {
+                        logger.debug("{}: Thing is disabled, skip initialization", thingName);
+                        return;
+                    }
```

Comment:
What's the purpose of this code? Is there any case when the Thing is disabled and we come here?